ui/plugins/talos/main.html

```diff
@@ -10,7 +10,14 @@
       </ul>
     </li>
   </ul>
-
+  <ul ng-show="perf_job_detail">
+    <li>Perfherder:
+      <ul ng-repeat="line in perf_job_detail | toArray | orderBy: '$key'">
```

Comment:
orderBy: '$key' should be orderBy: 'suite'. $key isn't defined for the perfherder array.